infra/docker-compose.yml

```diff
@@ -2,7 +2,7 @@
 version: '3'
 services:
   gotrue: # Signup enabled, autoconfirm off
-    image: supabase/gotrue:latest
+    image: supabase/auth:v2.129.0
```

Comment:
why do we not wanna pin against `latest`?

Reply:
It's not published. We need to see what we do with this https://github.com/supabase/gotrue/blob/master/.github/workflows/publish.yml#L36